lib/nylas/resources/events.rb

```diff
@@ -94,5 +94,18 @@ def send_rsvp(identifier:, event_id:, request_body:, query_params:)
         request_body: request_body
       )
     end
+
+    # Returns a list of recurring events, recurring event exceptions, and single events
+    # from the specifiedcalendar within a given time frame.
```

Comment:
```suggestion
    # Returns a list of recurring events, recurring event exceptions, and single events
    # from the specified calendar within a given time frame. This is useful when you 
    # want to import, store, and synchronize events from the time frame to your application 
```

Because the endpoint is such a strange name, I want to provide context as to why it is called import.